### ISSUE TITLE
Build dev.templates.tests.full_20180114.1 failed

### ISSUE THREAD

**Issue by wasteam** [Microsoft/WindowsTemplateStudio#1686]
## Build dev.templates.tests.full_20180114.1
- **Build result:** `failed`
- **Build queued:** 1/14/2018 6:41:51 PM
- **Build duration:** 3.36 minutes
### Details
Build [dev.templates.tests.full_20180114.1](https://winappstudio.visualstudio.com/web/build.aspx?pcguid=a4ef43be-68ce-4195-a619-079b4d9834c2&builduri=vstfs%3a%2f%2f%2fBuild%2fBuild%2f24673) failed

+ Unhandled: Not found SourceFolder: E:\UIT
+ Process completed with exit code 1.

Find detailed information in the [build log files](https://uwpctdiags.blob.core.windows.net/buildlogs/dev.templates.tests.full_20180114.1_logs.zip)

**Comment by ralarcon** [comment]
Actual failure is due to the UI tests not passing. Needs to review / investigate...

**Comment by sibille** [comment]
Looking at the build server there was a junction pointing to a non existing folder, which caused the templates to be unavailable. I reviewed the process of junction construction, which is correct, in other build servers junctions are correctly created. Assuming it is a problem of a corrupt file on the build server I tidied up the template cache and working folder, next build was successful on that server, so closing this for now.
